.github/workflows/ci_python.yml

```diff
@@ -0,0 +1,175 @@
+# This file is autogenerated by maturin v1.2.0
+# To update, run
+#
+#    maturin generate-ci github --pytest --zig -o ../.github/workflows/ci_python.yml
+#
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
```

Comment:
I think each of these flows will need to run the `python-semantic-release` tool to update the version in the local file to reflect the release we want to publish. Not sure enough about Python versioning yet.